README.md

```diff
@@ -231,6 +232,8 @@ The database is assumed to be a PostgreSQL database.
 
 This could be spun up by PostgreSQL or from Amazon RDS, for example.
 
+One thing to note is that our database connection's ssl mode is `verify-full` therefore your db instance must have a root certificate on its server, and you must have a copy of a leaf leaf certificate signed by the root certificate in a location accessible by this flask webapp, specified by `DB_SSL_ROOT_CERTIFICATE`.
```

Comment:
PR looks good, just a minor change to remove the duplicate word "leaf" from the README

Reply:
```suggestion
One thing to note is that our database connection's ssl mode is `verify-full` therefore your db instance must have a root certificate on its server, and you must have a copy of a leaf certificate signed by the root certificate in a location accessible by this flask webapp, specified by `DB_SSL_ROOT_CERTIFICATE`.
```